times_reader/transforms.py

```diff
@@ -888,6 +888,13 @@ def process_user_constraint_table(
                 df[colname] = [None] * nrows
         table = replace(table, dataframe=df)
 
+        # Fill missing regions using defaults (if specified)
+        regions_lists = [x for x in table.uc_sets.keys() if x.upper().startswith("R")]
```

Comment:
Do all regions have to start with `R`? If so, and we don't already, perhaps we should check and enforce that somewhere?

Reply:
No, there can be two keys in that dictionary (`R_E` and `R_S`) that start with R.

Reply:
I need to think more about this, but in case neither `R_E` nor `R_S` is specified on the sheet, we might need to discard the table and alert the user about it.

Reply:
Thanks @BakytzhanSuleimenov! As far as I know, some of them are ignored, even if specified (i.e. the info is always derived). This is okay for now. We'll make it perfect later. 😃